contracts/src/BeefyClient.sol

```diff
@@ -212,9 +233,14 @@ contract BeefyClient {
         external
     {
         ValidatorSet memory vset;
+        uint16 numRequiredSignatures;
         if (commitment.validatorSetID == currentValidatorSet.id) {
+            numRequiredSignatures = currentValidatorSetCounters.get(proof.index);
+            currentValidatorSetCounters.set(proof.index, numRequiredSignatures.saturatingAdd(1));
             vset = currentValidatorSet;
         } else if (commitment.validatorSetID == nextValidatorSet.id) {
+            numRequiredSignatures = nextValidatorSetCounters.get(proof.index);
```

Comment:
```suggestion
            signatureUsageCount = nextValidatorSetCounters.get(proof.index);
```

Reply:
Addressed in https://github.com/Snowfork/snowbridge/pull/956/commits/3da2bef5c6b3b13b02a5f4cd45a63e375c8c39e9